### DISCUSSION TITLE
Github Projects: Card Color Highlight

### DISCUSSION THREAD

**Discussion by Mirex97** [community/community#42431] [Projects and Issues]
### Select Topic Area

Product Feedback

## Feature Request: Card Color Highlight
Add card color as a custom setting for each card that could be enabled and the color picked from a palette (or with hex) similarly to how you can define a color for each Label.

This would be helpful to highlight cards that are important and need extra attention. And would be nice as a personalization option.

### Example
![CardColorHighlight](https://i.imgur.com/R60mdrk.png)

**Comment by brhoff720** [comment]
Would also add I feel this would make timeline/Roadmap views easier to read/share when presenting virtually.

**Comment by thanwill** [comment]
Essa realmente seria uma boa sugestão. Ainda mais na visualização por roadmap.
